src/plotman/chia.py

```diff
@@ -0,0 +1,83 @@
+import functools
+
+import click
+from pathlib import Path
+
+
+class Commands:
+    def __init__(self):
+        self.by_version = {}
+
+    def register(self, version):
+        if version in self.by_version:
+            raise Exception(f'Version already registered: {version!r}')
+        if not isinstance(version, tuple):
+            raise Exception(f'Version must be a tuple: {version!r}')
+
+        return functools.partial(self._decorator, version=version)
+
+    def _decorator(self, command, *, version):
+        self.by_version[version] = command
+        # self.by_version = dict(sorted(self.by_version.items()))
+
+    def __getitem__(self, item):
+        return self.by_version[item]
+
+    def latest_command(self):
+        return max(self.by_version.items())[1]
+
+
+commands = Commands()
+
+
+@commands.register(version=(1, 1, 2))
+@click.command()
+# https://github.com/Chia-Network/chia-blockchain/blob/v1.1.2/LICENSE
+# https://github.com/Chia-Network/chia-blockchain/blob/v1.1.2/chia/cmds/plots.py#L39-L83
```

Comment:
Here is the present indication of which code came from `chia-blockchain` and where it came from.  It does mark the specific lines copied in the link, though there could be a 'start of copy' and 'end of copy' mark in our code to be more explicit, if you want.  There should be no need to 'update' this specific code.  It is exactly for `chia-blockchain==1.1.2` and that "should" never change.  But, we will want to add newer (and possibly older) versions as well.  I mean...  there's already 1.1.3 and this hasn't even been merged.

So...  Yeah, I guess some maintenance documentation could be good.  I'll make a commit for adding 1.1.3 and some commentary somewhere can link to that.

On the one hand it is too bad that I'm going with copying the code.  But, the big flexibility it brings is the ability to have multiple versions available.  There are too many ways for programs to be distributed to bank on 'plotman should use the chia in its own env' unless we just mandate everyone do that.  But if anything, we probably ought to be going the other way and distributing plotman as a regular old program instead of exclusively as a Python package.  (this isn't a big deal to do, at least in a preliminary manner)  One of the issues that has come up a few times recently is macOS users that are running chia from the `.dmg`.  So, all in all this isn't that filthy.  Using the chia in the env wouldn't necessarily get us the proper parsing code for whatever random process we are looking at anyways.  So...  meh.  I think this isn't exactly the same as other cases of just copying code.